frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
 14. Once the role has been created record the role ARN as we'll need to use it later.
 
+#### System Role
+
+1. The example below assumes you've created the permission boundary policy above in the [App Role](App Role) section and named the policy `mlspace-project-user-permission-boundary`. If you've named it something different
+you'll need to update the `"iam:PermissionsBoundary` condition in the `iam:CreateRole` statement below.
+2. After the permission boundary policy has been created you are now ready to create the application
+policy. From the IAM Service page click "Policies" on the left hand side.
+3. Create a new policy using the JSON editor and paste the following in the text area (after replacing the placeholder variables):
+
+```JSON
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "kms:RetireGrant",
+            "Resource": "arn:{AWS_PARTITION}:kms:{AWS_REGION}:{AWS_ACCOUNT}:key/{MLSPACE_KMS_KEY_ID}",
+            "Effect": "Allow"
+        },
+        {
+            "Action": "logs:FilterLogEvents",
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:PassRole",
+                "iam:ListRoleTags"
+            ],
+            "Resource": [
+                "{EMR_DEFAULT_ROLE_ARN}",
+                "{EMR_EC2_INSTANCE_ROLE_ARN}",
+                "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/{MLSPACE_APP_ROLE_NAME}"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "dynamodb:GetItem",
+                "dynamodb:PutItem",
+                "dynamodb:Scan",
+                "dynamodb:DeleteItem",
+                "dynamodb:Query",
+                "dynamodb:UpdateItem"
+            ],
+            "Resource": [
+                "arn:{AWS_PARTITION}:dynamodb:{AWS_REGION}:{AWS_ACCOUNT}:table/mlspace-*"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": "ec2:AuthorizeSecurityGroupIngress",
+            "Resource": "arn:{AWS_PARTITION}:ec2:{AWS_REGION}:{AWS_ACCOUNT}:security-group/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "ec2:DescribeInstances",
+                "ec2:DescribeRouteTables",
+                "ec2:DescribeInstanceTypeOfferings",
+                "ec2:DescribeVpcEndpoints"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "s3:List*",
+                "s3:Get*",
+                "s3:DeleteObject",
+                "s3:PutBucketNotification"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:s3:::*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:UpdateNotebookInstanceLifecycleConfig"
```

Comment:
Probably also need describe and delete here
```
'sagemaker:DeleteNotebookInstanceLifecycleConfig',
'sagemaker:DescribeNotebookInstanceLifecycleConfig',
```

Looks like they are in a separate block below. "Resource" and "Effect" seem to be the same with no condition, so I think they can be consolidated.

Reply:
That's in another section for this policy, I'll combine them (not sure why they were separate in the app policy I originally copied)